registration/urls.py

```diff
@@ -14,10 +14,17 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-
 from django.contrib import admin
 from django.urls import path
+from django.views.generic.base import TemplateView
+#from frontend import views
+#from frontend.views import base_view, test_view
+
+
+#For now we're using function based views, but we might change to class based views later on if needed. We might need it for registration api users.?
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
-]
+    path('admin/', admin.site.urls), # This is the admin page for the project.
+    path('', TemplateView.as_view(template_name='base.html'), name='home'),  # Root URL -> base.html
```

Comment:
I don't think we should be using the TemplateView here, I'd rather have a function view with the render macro.
Easier to pass database data to the template at a later point then.

Reply:
That makes sense. I will switch to creating a function that renders a template in views.py instead.